src/components/navigation/DesktopLinks.jsx

```diff
@@ -0,0 +1,27 @@
+import Logout from './Logout';
```

Comment:
please correct the link from capital L to small case l

Reply:
Thanks for the feedback, @stephenkati. The link is correct, following the best practices for naming conventions. Please look at the [documentation](https://react.dev/learn) and these articles.

- [LinkedIn article](https://www.linkedin.com/pulse/react-js-naming-convention-kristiyan-velkov)
- [Medium article](https://medium.com/@smail.oubaalla/how-to-name-your-react-component-conventions-b8daf3abc574)

Reply:
the naming convention maybe right but the file you are refering to does not exists

Reply:
Ok. So I did rename the logout link component name, to begin with a capital letter (`logout.jsx` to `Logout.jsx`) but it seems like that change was not registered by `git`.

I have renamed it to `LogoutLink.jsx` and now the change has been registered by `git`.